tests/stdlib/tmath.nim

```diff
@@ -310,6 +310,11 @@ template main =
     doAssert signbit(-0.1)
     doAssert not signbit(0.1)
 
+    doAssert not signbit(Inf)
+    doAssert signbit(-Inf)
+    doAssert not signbit(NaN)
+    doAssert not signbit(-NaN)
```

Comment:
that's a because of a bug in `-` that we'll eventually have to fix, refs https://github.com/timotheecour/Nim/issues/499

not a blocker though, we'll just change this assert (and similar ones in this file)

Reply:
case in point: CI fails

2021-01-06T02:17:10.5225810Z ../../lib/system/fatal.nim(53, 5) Error: unhandled exception: tmath.nim(316, 5) `not signbit(-NaN)`  [AssertionDefect]

Reply:
It works in C/JS backend?